<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuningDialog</class>
 <widget class="QDialog" name="TuningDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuning</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tuningNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sharpsLabel">
       <property name="text">
        <string>Sharps:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="sharpsCheckBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="musicNotationOffsetLabel">
       <property name="text">
        <string>Music Notation Offset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="notationOffsetSpinBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="numberOfStringsLabel">
       <property name="text">
        <string>Number of Strings:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="numStringsSpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="tuningLabel">
       <property name="text">
        <string>Tuning:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="string8"/>
       </item>
       <item>
        <widget class="QComboBox" name="string7"/>
       </item>
       <item>
        <widget class="QComboBox" name="string6"/>
       </item>
       <item>
        <widget class="QComboBox" name="string5"/>
       </item>
       <item>
        <widget class="QComboBox" name="string4"/>
       </item>
       <item>
        <widget class="QComboBox" name="string3"/>
       </item>
       <item>
        <widget class="QComboBox" name="string2"/>
       </item>
       <item>
        <widget class="QComboBox" name="string1"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="presetLabel">
       <property name="text">
        <string>Preset:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="presetComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="capoLabel">
       <property name="text">
        <string>Capo:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="capoSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TuningDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TuningDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
